src/components/TitleBarEditorSwitcher.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+タイトルバーに配置される、エディタを切り替えるボタン
+-->
+
+<template>
+  <!-- FIXME: 画面サイズが小さくなると表示が崩れるのを直す -->
+  <!-- NOTE: デザインしづらいからQBtnかdivの方が良い -->
+  <q-btn-toggle
+    :model-value="nowEditor"
+    unelevated
+    dense
+    toggle-color="primary"
+    :options="[
+      { label: 'トーク', value: 'talk' },
+      { label: 'ソング', value: 'song' },
+    ]"
+    @update:model-value="gotoLink"
+  />
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+import { useRouter } from "vue-router";
+
+const router = useRouter();
+const nowEditor = computed<"talk" | "song">(() => {
+  const path = router.currentRoute.value.path;
+  if (path === "/talk") return "talk";
+  if (path === "/song") return "song";
+  window.electron.logWarn(`unknown path: ${path}`);
+  return "talk";
+});
+
+const gotoLink = (editor: "talk" | "song") => {
+  router.push("/" + editor);
+};
+</script>
+
+<style scoped lang="scss">
+@use '@/styles/variables' as vars;
+@use '@/styles/colors' as colors;
+.q-btn-group {
+  :deep(.q-btn) {
+    padding-left: 0.75rem;
+    padding-right: 0.75rem;
+  }
+
+  // 選択されているボタンの文字を太字にする
+  :deep(.q-btn[aria-pressed="true"]) {
+    span {
+      font-weight: 700;
+      color: colors.$display !important;
```

Comment:
```suggestion
      color: colors.$toolbar-button-display !important;
```
こちらの方が、ダークテーマにした際の視認性が向上しそうでした...!

Reply:
ダークモードのこと完全に忘れてました、ありがとうございます！！
（display-on-primary`が意図としてあってそうだったのdeこっちにしようと思います！）